api/.env

```diff
@@ -69,6 +69,10 @@ OAUTH_CEVIDB_CLIENT_ID="raT1QFf6TOQzpn3yVH-My6YLrmsvOrfMhYypxzjPMWk"
 OAUTH_CEVIDB_CLIENT_SECRET="fTxMrzjBn3gPGg3eB0bNMmjRqg4ccs3_su7CaTXtljE"
 OAUTH_CEVIDB_BASE_URL="https://cevi.puzzle.ch"
 
+OAUTH_JUBLADB_CLIENT_ID="WrKABq7GwmC6h1F0W73OGX_fOTHWWXnKXfrPMHOdQWY"
+OAUTH_JUBLADB_CLIENT_SECRET="oQ164RDMIAocL6PhmCoeT1Ymcg-7WcOJZdxCnIph5gM"
```

Comment:
Assuming these are the test secrets, right?

Reply:
Yes, they work with the integration environment of jubla.db.
@simfeld have you already added the prod secrets on github?

Reply:
Exactly, these are the secrets for jubladb integration. I don't have the prod secrets yet. I will let them know once it's deployed to dev and ask them to pass me the prod secrets. I need to add the test secrets on github as well for the dev deployment, right?

Reply:
Yes, that's right.